actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -220,8 +220,9 @@ def link_to(name = nil, options = nil, html_options = nil, &block)
       # +:form_class+ option within +html_options+. It defaults to
       # <tt>"button_to"</tt> to allow styling of the form and its children.
       #
-      # The form submits a POST request by default. You can specify a different
-      # HTTP verb via the +:method+ option within +html_options+.
+      # The form submits a POST request by default if the object is not persisted;
+      # conversely, if the object is persisted, it will submit a PATCH request.
+      # You can specify a different HTTP verb via the +:method+ option within +html_options+.
```

Comment:
We try to avoid using "you" in the documentation:
https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#pronouns

Maybe something like:
```suggestion
      # The form submits a POST request by default if the object is not persisted;
      # conversely, if the object is persisted, it will submit a PATCH request.
      # To specify a different HTTP verb use the +:method+ option within +html_options+.
```

Reply:
Thanks for the guidance! I'll revise the documentation accordingly to adhere to the style guidelines.